Samples/WinMLSamplesGallery/TransformAsync/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.AI.DirectML" version="1.8.0" targetFramework="native" />
```

Comment:
>1.8.0 [](http://example.com/codeflow?start=47&length=5)

Let's update to the current release:

Microsoft.AI.DirectML 1.9.0
Microsoft.AI.MachineLearning 1.12.1